### PR TITLE
Implemented Solr Schema Validation and Storage

### PR DESCRIPTION
**Pull request by skdhitman**
- Schema Validation ✔
- Schema Storage ✔
- Unit testing ✔

### CODE REVIEW

File: src/main/java/com/solr/clientwrapper/domain/service/SolrSchemaStorageService.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	
+	private static String DEFAULT_COLLECTION = "techproducts";
+	
+	String urlString = "http://localhost:8985/solr/";
+	
+	SolrClient solr = new HttpSolrClient.Builder(urlString+DEFAULT_COLLECTION).build();
+	
+	@Override
+	public String validateSchema() {
+		System.out.println("!!!!!!!!!!11 %%%%%%%%%%%%%%%% 11!!!!!!!!!!!!!!");
```
Comment (karthik261099) [inline src/main/java/com/solr/clientwrapper/domain/service/SolrSchemaStorageService.java]: In the entire code don't use System.out.println rather use loggers

File: src/main/java/com/solr/clientwrapper/domain/dto/solr/SolrSchemaDTO.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode
+public class SolrSchemaDTO {
+
+	String tableName;
+	String name;
+	SolrFieldDTO [] attributes;
+
```
Comment (karthik261099) [inline src/main/java/com/solr/clientwrapper/domain/dto/solr/SolrSchemaDTO.java]: A list would be better than array

File: src/main/java/com/solr/clientwrapper/domain/service/SolrSchemaService.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import com.solr.clientwrapper.domain.dto.solr.SolrSchemaDTO;
+import com.solr.clientwrapper.domain.port.api.SolrSchemaServicePort;
+
+@Service
+@Transactional
+public class SolrSchemaService implements SolrSchemaServicePort {
+
+	private final Logger log = LoggerFactory.getLogger(SolrSchemaService.class);
+	
+	HttpSolrClient solrClient = new HttpSolrClient.Builder("http://localhost:8983/solr").build();
```
Comment (karthik261099) [inline src/main/java/com/solr/clientwrapper/domain/service/SolrSchemaService.java]: The url should be taken from the properties file. Please check Collection_CRUD branch for more details

File: src/main/java/com/solr/clientwrapper/rest/SolrSchemaResource.java [hunk 4]
```diff
@@ -41,50 +35,60 @@
...
+		SolrSchemaDTO solrSchemaDTO = createSolrSchema.create(
+				newSolrSchemaDTO.getTableName(), 
+				newSolrSchemaDTO.getName(),
+				newSolrSchemaDTO);
+		return ResponseEntity.status(HttpStatus.OK).body(solrSchemaDTO);
 	}
 
-	@DeleteMapping("/delete/{name}")
+	@DeleteMapping("/delete/{tableName}/{name}")
 	@Operation(summary = "/delete-schema", security = @SecurityRequirement(name = "bearerAuth"))
```
Comment (karthik261099) [inline src/main/java/com/solr/clientwrapper/rest/SolrSchemaResource.java]: Delete method should take data from request body and not path parameter

File: src/main/java/com/solr/clientwrapper/rest/SolrSchemaResource.java [hunk 5]
```diff
@@ -41,50 +35,60 @@
...
+			@PathVariable String name) {
 		log.debug("Schema Delete");
-		deleteSolrSchema.delete(name);
-		return ResponseEntity.noContent().build();
+		deleteSolrSchema.delete(tableName, name);
+		return ResponseEntity.ok().build();
 	}
 
-	@PutMapping("/update")
+	@PutMapping("/update/{tableName}/{name}")
```
Comment (karthik261099) [inline src/main/java/com/solr/clientwrapper/rest/SolrSchemaResource.java:67]: Put method should take data from request body and not path variable

File: src/main/java/com/solr/clientwrapper/domain/service/SolrSchemaStorageService.java (hunk 1 above)
Comment (skdhitman) [reply src/main/java/com/solr/clientwrapper/domain/service/SolrSchemaStorageService.java]: Yes, that method was just to tests some stuffs. Had to remove at last, will do that.

File: src/main/java/com/solr/clientwrapper/domain/dto/solr/SolrSchemaDTO.java (hunk 2 above)
Comment (skdhitman) [reply src/main/java/com/solr/clientwrapper/domain/dto/solr/SolrSchemaDTO.java]: Actually, this is the way Kirti Ma'am probably communicated with Mangesh, it's his part of code. I also thought the same, but.., will consult with them.

File: src/main/java/com/solr/clientwrapper/domain/service/SolrSchemaService.java (hunk 3 above)
Comment (skdhitman) [reply src/main/java/com/solr/clientwrapper/domain/service/SolrSchemaService.java]: Yeah, that code has been replaced now. So, done.

File: src/main/java/com/solr/clientwrapper/rest/SolrSchemaResource.java (hunk 4 above)
Comment (skdhitman) [reply src/main/java/com/solr/clientwrapper/rest/SolrSchemaResource.java]: Ok, will talk about it with Mangesh. Web layer is implemented by him.

File: src/main/java/com/solr/clientwrapper/rest/SolrSchemaResource.java (hunk 5 above)
Comment (skdhitman) [reply src/main/java/com/solr/clientwrapper/rest/SolrSchemaResource.java:76]: Will talk to Mangesh about it.

**Pull request merged**